config/paths-common.conf

```diff
@@ -61,3 +61,6 @@ dovecot_log = %(syslog_mail_warn)s
 solidpop3d_log = %(syslog_local0)s
 
 mysql_log = %(syslog_daemon)s
+
+# Directory with ignorecommand scripts
+ignorecommands_dir = /etc/fail2ban/filter.d/ignorecommands
```

Comment:
My thought of having it here was so it could be referenced in the jail and changed in one place (not counting setup.py). If three more jails were added that use it, it would reduce the instances to change if need be.